source/containers.tex

```diff
@@ -16086,6 +16093,1215 @@
+
+    iterator upper_bound(const key_type& x);
+    const_iterator upper_bound(const key_type& x) const;
+    template<class K> iterator upper_bound(const K& x);
+    template<class K> const_iterator upper_bound(const K& x) const;
+
+    pair<iterator, iterator> equal_range(const key_type& x);
+    pair<const_iterator, const_iterator> equal_range(const key_type& x) const;
+    template<class K>
+      pair<iterator, iterator> equal_range(const K& x);
+    template<class K>
+      pair<const_iterator, const_iterator> equal_range(const K& x) const;
+
+    friend bool operator==(const flat_multiset& x, const flat_multiset& y);
+
+    friend @\exposconcept{synth-three-way-result}@<value_type>
+    operator<=>(const flat_multiset& x, const flat_multiset& y);
+
+    friend void swap(flat_multiset& x, flat_multiset& y) noexcept
+      { x.swap(y); }
+
+  private:
+    container_type @\exposid{c}@;       // \expos
+    key_compare @\exposid{compare}@;    // \expos
+  };
+
+  template<class InputIterator, class Compare = less<@\exposid{iter-value-type}@<InputIterator>>>
+    flat_multiset(InputIterator, InputIterator, Compare = Compare())
+      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, @\exposid{iter-value-type}@<InputIterator>, Compare>;
+
+  template<class InputIterator, class Compare = less<@\exposid{iter-value-type}@<InputIterator>>>
+    flat_multiset(sorted_equivalent_t, InputIterator, InputIterator, Compare = Compare())
+      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, @\exposid{iter-value-type}@<InputIterator>, Compare>;
+
+  template<ranges::input_range R, class Compare = less<ranges::range_value_t<R>>,
+          class Allocator = allocator<ranges::range_value_t<R>>>
+    flat_multiset(from_range_t, R&&, Compare = Compare(), Allocator = Allocator())
+      -> flat_multiset<ranges::range_value_t<R>, Compare>;
+
+   template<ranges::input_range R, class Allocator>
+     flat_multiset(from_range_t, R&&, Allocator)
+       -> flat_multiset<ranges::range_value_t<R>, less<ranges::range_value_t<R>>>;
+
+  template<class Key, class Compare = less<Key>>
+    flat_multiset(initializer_list<Key>, Compare = Compare())
+      -> flat_multiset<Key, Compare>;
+
+  template<class Key, class Compare = less<Key>>
+  flat_multiset(sorted_equivalent_t, initializer_list<Key>, Compare = Compare())
+      -> flat_multiset<Key, Compare>;
+
+  template<class Key, class Compare, class KeyContainer, class Allocator>
+    struct uses_allocator<flat_multiset<Key, Compare, KeyContainer>, Allocator>
+      : bool_constant<uses_allocator_v<KeyContainer, Allocator>> { };
+}
+\end{codeblock}
+
+\rSec3[flat.multiset.ctor]{Constructors}
+
+\indexlibraryctor{flat_multiset}%
+\begin{itemdecl}
+flat_multiset(container_type cont);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes \exposid{c} with \tcode{std::move(cont)},
+value-initializes \exposid{compare}, and
```

Comment:
Replace all `\exposid{compare}` with `\tcode{compare}`.

Reply:
Why so? `compare` is a private, exposition-only member, isn't it?

Reply:
See https://github.com/cplusplus/draft/pull/5696#discussion_r939344294.